packages/storybook/stories/components/divider/divider.stories.ts

```diff
@@ -0,0 +1,107 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { ODS_DIVIDER_COLOR, ODS_DIVIDER_COLORS, ODS_DIVIDER_SPACING, ODS_DIVIDER_SPACINGS } from '@ovhcloud/ods-components';
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-divider';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
+import { html } from 'lit-html';
+
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/Content/Divider',
+  component: 'ods-divider',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (arg) => html`
+<ods-divider class="my-divider-demo"
+             color="${arg.color}"
+             spacing="${arg.spacing}">
+</ods-divider>
+<style>
+  .my-divider-demo::part(divider) {
+    ${arg.customCss}
+  }
+</style>
+  `,
+  argTypes: orderControls({
+    color: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: ODS_DIVIDER_COLOR.light },
+        type: { summary: `Type string: ${ODS_DIVIDER_COLORS}` }
+      },
+      // todo when options are defined, type select should be automatically inferred but not working?
+      control: { type: 'select' },
+      options: ODS_DIVIDER_COLORS,
+    },
+    spacing: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: ODS_DIVIDER_SPACING._2 },
+        type: { summary: `Type string from: ${ODS_DIVIDER_SPACINGS}` }
+      },
+      control: { type: 'select' },
+      options: ODS_DIVIDER_SPACINGS,
+    },
+    customCss: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' }
+      },
+      control: 'text',
+      description: 'Set a custom style properties. Example: "height: 100px; color: #008000;"',
+    }
+  }),
+  args: {
+    color: ODS_DIVIDER_COLOR.light,
+    spacing: ODS_DIVIDER_SPACING._2,
+  },
+};
+
+export const Default: StoryObj = {
+  tags: ['isHidden'],
+  render: () => html`
+<ods-divider></ods-divider>
+  `,
+};
+
+
+export const Color: StoryObj = {
+  tags: ['isHidden'],
+  render: () => html`
+<ods-divider color="light"></ods-divider>
```

Comment:
please add spacing on each, otherwise the rendering is not easy to see

Reply:
Done